helper/src/components/portalnav.js

```diff
@@ -118,6 +119,14 @@ export default function PortalNav({ config }) {
   const [tabValues, setTabValues] = useState(initTabValues(selected, defaults, true))
   // Field Selections - Used to keep track of the last FieldSelections monitored by App Insights to prevent logging the same entry continuously
   const [lastAIUpdated, setLastAIUpdated] = useState ({tab: null, field: null})
+  const [show, setShow] = useState(false)
+  const [previewLink, setPreviewLink] = useState("")
+
+  const showPreviewModal = (e,previewLink) => {
+    setShow(current => !current);
```

Comment:
Not sure what 'current' is is this needed?

Reply:
good catch, this should be the `show` status toggler